extensions/adobe/experience/intelligentServices/profile-journeyai-sendtimeoptimization.schema.json

```diff
@@ -32,7 +32,7 @@
                   "type": "object",
                   "properties": {
                     "https://ns.adobe.com/experience/intelligentServices/sendTimeScoreForClick": {
-                      "title": "Send Time Score for Click",
+                      "title": "Send Time Score for Click (to be deprecated)",
```

Comment:
Please add "meta:status": "deprecated" to the property to deprecate the property.

Reply:
Hi @mjasrotia, as I mentioned in the Jira ticket, we plan to keep these "to be deprecated" fields co-existing with the newly added fields for a while for old data preserving and new score validation. We will mark the fields as deprecated later. Is this allowed?

Reply:
@spengadobe does this mean data for both properties will be populated in the dataset ? if it is just to access old data , you can still access old data set with old structure even when you deprecate these properties , its just that you will not be able to populate new data set using deprecated properties . Hope this makes sense

Reply:
Got it. I will change the code then. Thanks!